presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -50,13 +52,23 @@ public final void destroy()
     }
 
     @Test
-    public void testDropTable()
+    public void testTableLifecycleWithNamesContainingUnderscores()
             throws Exception
     {
-        assertQueryTrue("CREATE TABLE test_drop AS SELECT 123 x");
-        assertTrue(queryRunner.tableExists(getSession(), "test_drop"));
+        Session session = testSessionBuilder()
+                .setCatalog("mysql")
+                .setSchema("test_database")
+                .build();
 
-        assertQueryTrue("DROP TABLE test_drop");
-        assertFalse(queryRunner.tableExists(getSession(), "test_drop"));
+        String tableName = "test_table_with_name_containing_underscores";
+        assertFalse(queryRunner.tableExists(session, tableName));
+
+        assertQuery(session, "CREATE TABLE " + tableName + " AS SELECT 123 x", "SELECT true");
```

Comment:
Actually, `assertQueryTrue` should be used for things like `DROP TABLE` that do not return a result. `CREATE TABLE AS` returns the row count (just like `INSERT` or `DELETE`), so it should actually be compared to `SELECT 1`. I'm surprised `SELECT true` works here (I think that's a bug in the testing framework).